### ISSUE TITLE
"ceedling new blah" fails when the current dir contains a project.yml file

### ISSUE THREAD

**Issue by hmijail** [ThrowTheSwitch/Ceedling#61]
I think ceedling should fail more gracefully than just "rake aborted! Don't know how to build task 'new' (see --tasks)".

**Comment by mvandervoord** [comment]
Good point.

What kinds of things do you feel it should display? A list of commands acceptable commands? Should it attempt to guess what you meant?

**Comment by hmijail** [comment]
Not sure, I still know too little about the whole workflow with ceedling. For context, my use case was probably absurd (I did "ceedling new blah" inside of a dir created by a previous "ceedling new"), but this apparent nonsense error sent me chasing my tail for a while. (I had just upgraded the ceedling gem, so I feared that the "ceedling new" command had been somehow removed or botched)

I guess that the root problem here is that now ceedling delegates to rake too soon if project.yml exists, which probably isn't a good idea anyway: if the rakefile did actually contain a "new" target, suddenly ceedling would be acting in an unexpected way. Ceedling should rather filter its own commands before delegating to rake.
